app/utils.py

```diff
@@ -89,47 +101,31 @@ def choose_layer_style(n_clicks):
 
 # ----------------------------------------------------------------------------------------------------------------------
 # Information box
-# The following block is used to build the box in the top-right corner and fill it in with the relevant information.
-
-def build_info_box(feature=None):
-    """
-    This function creates the HTML components of the information box.
-
-    It takes as argument the polygon hovered by the user in the departments GeoJSON object, if any.
-
-    It returns:
-
-    - the title of the information box and the name of the hovered when relevant;
-    - the title of the information box and a generic message when there is no department being hovered.
-    """
-    header_dept = [html.H4('Département sélectionné :')]
+# The following block is used to build the map filters in the bottom left corner
 
-    if feature:
-        return header_dept + [html.B(feature['properties']['nom'])]
+def build_map_filters(feature=None):
 
-    # If no object is hovered, it just returns a standard statement
-    return header_dept + [html.P('Faites glisser votre curseur sur un département')]
+    return build_layer_style_button()
 
 
-def build_info_object(map_type):
+def build_filters_object(map_type):
     """
-    This function builds upon the build_info_box function defined above.
+    This function builds upon the build_map_filters function defined above.
 
     It takes as input the type of map considered (either 'alerts' or 'risks') and returns
-    an information box object located in the top-right corner of the map with a relevant id.
+    map filters of the map with a relevant id.
     """
     if map_type == 'alerts':
         object_id = 'alerts_info'
```

Comment:
```suggestion
        object_id = 'alerts_filter_button'
```

Maybe we could adapt the ID of the object since it is not an information box anymore. Here is a proposition but it may not be the most appropriate one.

Reply:
Indeed, done !

Reply:
Done